### DISCUSSION TITLE
After Deploying and SEO ranking my site showing title "vercel.app"

### DISCUSSION THREAD

**Discussion by hardikd-zignuts** [vercel/community#4799] [Help]
### Summary

After deploying my website on Vercel, I've noticed that the SEO title displayed in search results is `vercel.app` instead of the intended `DragNest`. This issue is impacting the visibility and branding of my website. I need assistance in resolving this problem to ensure that the correct title is displayed in search engine results.

I would appreciate any guidance or solutions to rectify this issue and improve the SEO ranking of my website.

### Example

https://dragnest.vercel.app/

### Steps to Reproduce

SITE:- https://dragnest.vercel.app/dashboard/
![image](https://github.com/vercel/community/assets/124128351/f8794742-402d-4804-be2e-2eaf56cf9b29)
![image](https://github.com/vercel/community/assets/124128351/4243b99d-15b0-4bdd-b0ac-6dc36d2cba81)

**Comment by hardikd-zignuts** [comment]
Could you please assist me, @vercel, @rauchg, @tomocchino, @TooTallNate, and @vvo?

**Comment by amyegan** [comment]
Hey @hardikd-zignuts. Setting up your site for optimal search results can be tricky. Google has some good resources covering SEO and search result appearance. These can help you:

- [Provide a site name to Google Search](https://developers.google.com/search/docs/appearance/site-names)
- [Influencing your title links in search results](https://developers.google.com/search/docs/appearance/title-link)
- [How to Write Meta Descriptions](https://developers.google.com/search/docs/appearance/snippet)
- [Search Engine Optimization (SEO) Starter Guide](https://developers.google.com/search/docs/fundamentals/seo-starter-guide)

**Comment by hardikd-zignuts** [comment]
Thanks @amyegan it would be very helpful, Thanks a lot
